src/Altinn.Platform/Altinn.Platform.Storage/Storage/appsettings.json

```diff
@@ -14,7 +14,7 @@
     "OrgStorageAccount": "{0}altinndevstrg01",
     "OrgStorageContainer": "{0}-dev-appsdata-blob-db",
     "OrgSasDefinition": "{0}devsasdef01",
-    "OrgPrivateBlobStorageEnabled": "false"
+    "AllowedSasTokenAgeHours": 8
```

Comment:
Removed OrgPrivateBlobStorageEnabled - intentional?

Reply:
Yes, removed intentionally. It isn't being used by any code. I will also remove it as an environment variable from the values.yml file for storage.